styles/github/catppuccin.user.css

```diff
@@ -1374,39 +1420,178 @@
     .iQkwAM {
       color: @subtext0;
     }
-    
+
     smt {
       background-color: @base;
       &:hover {
         background-color: @surface0;
       }
     }
-    
+
     .dtTTcd {
       background-color: @base;
       &.hoverable:hover {
         background-color: @surface0;
       }
     }
-    
+
     .pQcjM {
       border-color: @surface1;
       &.hoverable:hover {
         background-color: @surface0;
       }
     }
-    
+
     .eTSXcP {
       background-color: @red;
     }
-    
+
     .iGxiw {
       border-bottom-color: @surface1;
     }
+
+    .IZXbW,
+    .jIRWwO {
+      color: @subtext0;
+    }
+
+    .bCilEY {
+      color: @text;
+    }
+
+    .iRopIK {
+      background-color: @green;
+      color: @base;
+    }
+
+    .kSJuJl {
+      background-color: @mantle;
+    }
+
+    .hcHzsC,
+    .eUjaFk {
+      background-color: @base;
+      border-color: fade(@surface0, 40%);
+
+      &:disabled {
+        opacity: 75%;
+      }
+    }
+
+    .hiBNcM {
+      background-color: @base;
+    }
+
+    .gFvSyc {
+      box-shadow: @accent-color 0px 0px 0px 2px;
+    }
+
+    .fKIicM {
+      background-color: @base;
+    }
+
+    .bCilEY::after {
+      background-color: @accent-color;
+    }
+
+    .dyFCSd {
+      color: @subtext0;
+
+      & th {
+        background-color: @mantle;
+      }
+    }
+
+    .cnmWFZ:hover {
+      background-color: @base;
+    }
+
+    .bQnDDD {
+      color: @accent-color;
+    }
+
+    .hkrwZC,
+    .dQkdWu,
+    .eRSvPK,
+    .kKLDSN,
+    .jPuYNf {
+      background-color: @mantle;
+      color: @subtext0;
+    }
+
+    .fvMdhy {
+      background-color: @mantle;
+      color: @subtext0;
+      border-color: @base;
+    }
+
+    .OEbqN {
+      color: @surface2;
+    }
+
+    .dDkgrC .PRIVATE_TreeView-item-container,
+    .dDkgrC .PRIVATE_TreeView-item-visual {
+      color: @subtext0 !important;
+    }
```

Comment:
This isn't changing anything for me, where i think at least, could you show me what and where it is changing for you